microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md

```diff
@@ -0,0 +1,180 @@
+title: What's new in DevTools (Microsoft Edge 96)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/16/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# What's New in DevTools (Microsoft Edge 96)
+
+The following sections list the announcements from the Microsoft Edge DevTools team.  To try the latest features of DevTools and the Microsoft Edge DevTools extension for Visual Studio Code, read these announcements.  To stay up to date with the latest and greatest features in your developer tools, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) and [follow the Microsoft Edge DevTools team on Twitter](https://twitter.com/EdgeDevTools).
+
+If you are on Windows, Linux, or macOS, consider using the Canary preview channel of Microsoft Edge as your default development browser.  The preview channels give you access to the latest features of Microsoft Edge DevTools.
+
+
+<!-- ====================================================================== -->
+## New DevTools UI available (in preview)
+
+<!-- 1 -->
+
+<!-- Title: New DevTools UI available (in preview) -->
+<!-- Subtitle: A more minimal, modern UI is coming to Microsoft Edge DevTools. Enable the "Focus Mode" experiment to preview new UI features such a more compact toolbar that keeps DevTools uncluttered and better adapts to small window sizes. -->
+
+The Microsoft Edge DevTools team is experimenting with a new DevTools UI: **Focus Mode**, which provides a vertical Activity Bar for all of your tools.  This Activity Bar gives a quick view, to have two tools open simultaneously, and provides several customization and organization options to personalize your tools.
+
+To give this new UI a try in Microsoft Edge version 96, select **Settings** (![the Settings gear icon in DevTools](../../../media/settings-gear-icon-light-mode.png)) > **Experiments** > **Focus Mode**.
+
+Starting with Microsoft Edge version 96, the experiment's checkbox is labeled **Focus Mode** rather than **Focus Mode and DevTools Tooltips**.
+
+This is an experiment, so the UI may change and evolve in future versions of Microsoft Edge.  We're eager to hear your feedback about this new DevTools UI.  Send us your feedback by tweeting [@EdgeDevTools](https://twitter.com/edgedevtools).  Or, with the **Focus Mode** experiment turned on, at the bottom of the **Activity Bar**, select **Help** (![the Help icon in the Activity Bar in Focus Mode](../../../media/help-icon-of-focus-mode.png)) > **Feedback**, to show the **Send feedback** window.
+
+:::image type="content" source="../../media/2021/11/focus-mode.png" alt-text="Focus Mode, including the Activity Bar.":::
+
+<!-- video: see work item -->
+
+See also:
+*  [Group tools together in Focus Mode](../02/devtools.md#group-tools-together-in-focus-mode) in _What's New In DevTools (Microsoft Edge 90)_.
```

Comment:
I added this link.  Please compare the present section to the linked section.
@hxlnt @zoherghadyali

Reply:
This link should be removed because this functionality (tool grouping) no longer exists.